code/modules/clothing/masks/_masks.dm

```diff
@@ -8,6 +8,7 @@
 	var/modifies_speech = FALSE
 	var/mask_adjusted = 0
 	var/adjusted_flags = null
+	var/darkness_view = 2 // default
```

Comment:
What is this for? This doesn'T do anything as it is, as far as I can tell!

Reply:
i *think* masks don't have that variable by default, so while that specific line is irrelevant, it *can* be useful in case we wish to add another item that blinds the player (by setting the darkness_view = 1)

we can also move that to here, which would work too!

![image](https://github.com/user-attachments/assets/40f5d033-605f-460e-b3a5-193a5d7ea828)

Reply:
No point leaving unused variables for possible future code